common/types/testutil.go

```diff
@@ -2,14 +2,12 @@ package types
 
 import (
 	"math/rand"
-	"time"
 
 	"github.com/spacemeshos/go-spacemesh/signing"
 )
 
 // RandomBytes generates random data in bytes for testing.
 func RandomBytes(size int) []byte {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
curious. what motivates removing randomness here?

Reply:
Time on Windows appears to have low resolution. Because of it, subsequent calls to `rand.Seed(...)` can be given the same seed value. As a result, it returns exactly the same "random" value in subsequent calls.

I'm not sure why it was given an explicit seed in the first place.

Reply:
rand seems wonky in the codebase right now. some use directly from math/rand, some from go-spacemesh rand pkg.